sagemaker_processing/local_pyspark/local_pyspark_example.ipynb

```diff
@@ -0,0 +1,499 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run PySpark locally on SageMaker Studio\n",
+    "\n",
+    "This notebook shows you how to run PySpark code locally within a SageMaker Studio notebook. The dependencies are installed in the notebok, so you can run this notebook on any image/kernel, including BYO images. For this example, you can choose the Data Science image and Python 3 kernel."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# setup - install JDK\n",
+    "# you only need to run this once per kernel\n",
+    "%conda install openjdk -y"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# install PySpark\n",
+    "%pip install pyspark==3.2.0"
```

Comment:
```suggestion
    "%pip install pyspark==3.1.0"
```

Let's match the version of PySpark we'll run in the processing job

Reply:
updated to pyspark==3.1.1